explorer/src/entity/ChainlinkNode.ts

```diff
@@ -53,7 +66,7 @@ export const createChainlinkNode = async (
   name: string
 ): Promise<[ChainlinkNode, string]> => {
   const secret = generateRandomString(64)
-  const chainlinkNode = new ChainlinkNode(name, secret)
+  const chainlinkNode = ChainlinkNode.build(name, secret)
```

Comment:
Why change from a constructor to a static build function?

Reply:
Many ORMs, including this one and ActiveRecord, reserve a parameterless constructor for internal use. I wanted to stray away from enforcing a contructor(name, secret) when `typeorm` or a developer would want to create an instance from fields retrieved from the db or when only having an id, ie:

```
const clnode = new ChainlinkNode()
clnode.id = 123
jobrun.chainlinkNode = clnode
```

Their documentation alludes to that usage: https://typeorm.io/#/eager-and-lazy-relations
I also went down that route but reverted back to `chainlinkNodeId` on job run in the end. Still think this is a better practice.